### PR TITLE
components message index.js - cleaned a bit

### PR DESCRIPTION
**Pull request by ArteomBalanuta**
* idk, looks better

### CODE REVIEW

File: app/components/Message/index.js [hunk 1]
```diff
@@ -207,6 +86,162 @@ function Message({ extended, type, payload, user, msgForm }) {
   );
 }
 
+function InviteFromMe() {
```
Comment (raf924) [inline app/components/Message/index.js:89]: Shouldn't these new functions take `payload`, `user` and `msgForm` as arguments ?

Comment (marzavec) [reply app/components/Message/index.js:89]: This ^

**Comment by marzavec** [comment]
@ArteomBalanuta Have you tested this?

**Comment by ArteomBalanuta** [comment]
@marzavec I have not

File: app/components/Message/index.js (hunk 1 above)
Comment (ArteomBalanuta) [reply app/components/Message/index.js]: @marzavec @raf924 should be fine now

Comment (raf924) [reply app/components/Message/index.js]: Still not. Please test your code.
If you call `MyFunction(payload)` then you can't define `function MyFunction({payload})`. That means your function expects an object with the property `payload`.

Comment (ArteomBalanuta) [reply app/components/Message/index.js]: @raf924 tested, pushed.

Comment (ArteomBalanuta) [reply app/components/Message/index.js]: @marzavec done

Comment (ArteomBalanuta) [reply app/components/Message/index.js]: Fixed /whisper bug that displayed same message when whispered yourself

Comment (ArteomBalanuta) [reply app/components/Message/index.js]: WIP - will fix formatting

**Comment by marzavec** [comment]
Once the conflicts are resolve, please lint your code using "npm run lint"

**Comment by ArteomBalanuta** [comment]
@marzavec @raf924 reopened the pr with rebase and lint applied. Apologize for stretching you on this pr..

**Pull request closed without merging**